docs/Data Provider Nodes/chainlink-alarm-clock.md

```diff
@@ -55,7 +55,7 @@ contract ChainlinkAlarmClock is ChainlinkClient {
 ```
 
 <div class="remix-callout">
-  <a href="https://remix.ethereum.org/#gist=e71aab73726c5a2f99ee00c1a70cfef8&optimize=true&version=soljson-v0.6.12+commit.27d51765.js&evmVersion=null&runs=200" target="_blank" class="cl-button--ghost solidity-tracked">Deploy this contract using Remix ↗</a>
+  <a href="https://remix.ethereum.org/#version=soljson-v0.6.7+commit.b8d736ae.js&optimize=false&evmVersion=null&url=https://github.com/smartcontractkit/documentation/tree/main/_src/samples/DataProviders/AlarmClock.sol" target="_blank" class="cl-button--ghost solidity-tracked">Deploy this contract using Remix ↗</a>
```

Comment:
Shouldn't these remix URLs point to `./tree/main/_includes/` instead of `./tree/main/_src/`? Or am I missing a redirect of some kind in GitHub itself?

Reply:
Probably need to point to raw.github.com

Reply:
Ideal would be: docs.chain.link/samples/DataProviders/AlarmClock.sol